apps/studio/src/lib/editor/engine/overlay/index.ts

```diff
@@ -68,6 +68,12 @@ export class OverlayManager {
         let top = 0,
             left = 0;
         while (element && element !== ancestor) {
+            const transform = window.getComputedStyle(element).transform;
+            const matrix = new DOMMatrix(transform);
+
+            top += matrix.m42;
```

Comment:
Just curious what does this do?

Reply:
To obtain the web frame's position, we should use `getComputedStyle` to retrieve the transform values, which cannot be obtained by `offset` properties. Since the values are presented in matrix format, I utilized the `DOMMatrix` class for easier manipulation.

Reply:
Gotcha, I learned something new today. Thanks!